src/html/ui/community/invite.tpl.html

```diff
@@ -81,7 +81,11 @@
           <button type="button" ng-click="edit()">Change</button>
```

Comment:
wait, we allow editing the code on this page? how did i miss that? that shouldn't be the case. specifically for the case in which all members are allowed to access this screen, they should not be able to also change the code. you can code in just that specific case if you want but also just dropping editing from this screen completely is fine with me too.